docs/pkg-ps.8

```diff
@@ -0,0 +1,83 @@
+.\"
+.\" Redistribution and use in source and binary forms, with or without
+.\" modification, are permitted provided that the following conditions
+.\" are met:
+.\" 1. Redistributions of source code must retain the above copyright
+.\"    notice, this list of conditions and the following disclaimer.
+.\" 2. Redistributions in binary form must reproduce the above copyright
+.\"    notice, this list of conditions and the following disclaimer in the
+.\"    documentation and/or other materials provided with the distribution.
+.\"
+.\"
+.\"     @(#)pkg.8
+.\"
+.Dd May 30, 2024
+.Dt PKG-PS 8
+.Os
+.Sh NAME
+.Nm "pkg ps"
+.Nd list running processes using deleted binaries and libraries
+.Sh SYNOPSIS
+.Nm
+.Op Fl v
+.Pp
+.Nm
+.Op Cm --verbose
+.Sh DESCRIPTION
+.Nm
+List running processes using deleted binaries and libraries.
+This program must be run as root if any of these sysctl's are set:
```

Comment:
Did you mean the other way around? If `security.bsd.unprivileged_proc_debug` isn't set ( =0 ) then this program must be run as root.

Reply:
Fixed.